infrastructure/cdn-in-a-box/README.md

```diff
@@ -0,0 +1,41 @@
+# CDN-In-A-bOx (CIAO) - Notes
+
+## Build step
+So far, the build is done from this directory (/infrastructure/cdn-in-a-box/), using the following command:
+
+```bash
+docker-compose -f ./docker-compose.yml -f traffic_ops/docker-compose.yml -f traffic_ops_db/docker-compose.yml -f traffic_portal/docker-compose.yml up --build
```

Comment:
This might be easier to use if there was just a single docker-compose file with multiple services in it

Reply:
agree,  but during development,  it was easier to have them separated..